autorevision.sh

```diff
@@ -1029,7 +1029,7 @@ pathSegment() {
 	local depth="0"
 
 	if [ ! -z "${pathz}" ]; then
-		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ]; do
+		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ] && [ ! "${pathz}" = "C:" ]; do
```

Comment:
OK,  changing the code on lines to 1029-1041 to

	
        echo "Path Segment ${pathz}" > /dev/stderr
	    if [ ! -z "${pathz}" ]; then
		echo "1:  ${pathz}"
		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ] && [ ! "${pathz}" = "" ]; do
			echo "2: ${pathz}" > /dev/stderr
			pathz="$(dirname "${pathz}")"
			depth="$((depth+1))"		
		done

and putting   `set -v`   at the begging of the file gives the following output:

pathSegment "${svnPath}"
Path Segment C:/testAutorev/apache-pig
2: C:/testAutorev/apache-pig
dirname "${pathz}"
2: C:/testAutorev
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C:
dirname "${pathz}"
2: C: